app-server/app/src/main/scala/user/User.scala

```diff
@@ -105,55 +92,83 @@ object UserService {
     }
   }
 
-  def getUserById(id: java.util.UUID )(implicit database: Database): Future[Option[UsersRow]] = {
+  def createUser
+    (user: UsersRowCreate)
```

Comment:
Kind of feel like this line splitting is a little excessive - if we need to maybe just move the implicit parameters to the second line